src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
```

Comment:
The message 'Truck started working' is not very informative. It would be better to specify what type of work the Truck is performing to make the message more informative.